app/components/Settings/scheduler.jsx

```diff
@@ -0,0 +1,307 @@
+                <Column style={{ paddingRight: '24px' }}>
+                    <em
+                        style={{
+                            fontSize: 15
+                        }}
+                    >
+                        {`Runs every ${ms(run.refreshInterval * 1000, {
+                            long: true
+                        })}`}
+                    </em>
+                </Column>
+            </Row>
+        );
+    }
+}
+
+const rowStyle = {
+    justifyContent: 'flex-start',
+    borderBottom: '1px solid rgba(0, 0, 0, 0.05)',
+    padding: '16px 0px'
+};
+const boxStyle = { boxSizing: 'border-box', width: '50%' };
+
+const MetaPreview = props => {
+    if (!props.query) {
+        return null;
+    }
+    const [account, gridId] = props.query.fid.split(':');
+    const link = `${plotlyUrl()}/~${account}/${gridId}`;
+
+    return (
+        <Column style={{ width: '50%', background: 'white' }}>
+            <Row
+                className="sql-preview"
+                style={{
+                    padding: '32px',
+                    position: 'relative',
+                    justifyContent: 'flex-start'
+                }}
+            >
+                <h5 className="sql-preview" style={{ margin: 0, letterSpacing: '1px' }}>
+                    <SQL className="bold">{props.query.query}</SQL>
+                </h5>
+                <button
+                    onClick={props.onCloseBtnClick}
+                    style={{
+                        position: 'absolute',
+                        top: '16px',
+                        right: '16px',
+                        padding: '2px 4px'
+                    }}
+                >
+                    &times;
+                </button>
+            </Row>
+            <Column style={{ background: '#F5F7FB', padding: '32px' }}>
+                <Row style={rowStyle}>
+                    <div style={boxStyle}>Query</div>
+                    <div className="sql-preview" style={boxStyle}>
+                        <SQL>{props.query.query}</SQL>
+                    </div>
+                </Row>
+                <Row style={rowStyle}>
+                    <div style={boxStyle}>Update Frequency</div>
+                    <em style={boxStyle}>
+                        Runs every{' '}
+                        <b>
+                            {ms(props.query.refreshInterval * 1000, {
+                                long: true
+                            })}
+                        </b>
+                    </em>
+                </Row>
+                <Row style={rowStyle}>
+                    <div style={boxStyle}>Live Dataset</div>
+                    <Link href={link} style={boxStyle}>
+                        {link}
+                    </Link>
+                </Row>
+            </Column>
+        </Column>
+    );
+};
+
+MetaPreview.propTypes = {
+    onCloseBtnClick: PropTypes.func,
+    query: PropTypes.object
+};
+
+export const MetaPreviewModal = props => (
```

Comment:
Is there any need for having two components: `MetaPreview` and `MetaPreviewModal`?

If possible, I'd like to have a single component with a name that doesn't require to document the purpose of these components (how about `SchedulePreview`?).